examples/rotation/run_rotation.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
```

Comment:
This will need to be updated.

Reply:
Let's also not forget to update `CMakeLists.txt` to include the two new examples (w/ and w/o python).

Reply:
Done.